src/main/java/org/ibp/api/java/impl/middleware/dataset/DatasetServiceImpl.java

```diff
@@ -86,9 +86,10 @@
 @Transactional
 public class DatasetServiceImpl implements DatasetService {
 
-	private static final String LOCATION_ID_VARIABLE_NAME = "LOCATION";
+	static final String LOCATION_ID_VARIABLE_NAME = "LOCATION";
+	static final String LOCATION_ABBR_VARIABLE_NAME = "LOCATION ABBREVIATION";
 	private static final List<Integer> PROTECTED_VARIABLE_IDS =
-		Arrays.asList(TermId.TRIAL_INSTANCE_FACTOR.getId(), TermId.LOCATION_ID.getId());
+		Arrays.asList(TermId.TRIAL_INSTANCE_FACTOR.getId(), TermId.LOCATION_ID.getId(), TermId.LOCATION_ABBR.getId());
```

Comment:
Hi @cheryleigh, there's an issue with this change. Users can't remove the LOCATION_ABBR from the environments tab.

Reply:
Thank you for pointing this out, I initially thought it affects the import. Removed this change :)